src/UI/class-settings-page.php

```diff
@@ -76,23 +76,34 @@ public function run(): void {
 	}
 
 	/**
-	 * Enqueue all needed scripts for Parse.ly plugin settings page.
+	 * Enqueue all needed scripts and styles for Parse.ly plugin settings page.
 	 *
 	 * @param string $hook_suffix The current page being loaded.
 	 * @return void
 	 */
-	public function enqueue_settings_scripts( string $hook_suffix ): void {
+	public function enqueue_settings_assets( string $hook_suffix ): void {
 		if ( 'settings_page_parsely' === $hook_suffix ) {
 			wp_enqueue_media();
 
 			$admin_settings_asset = require plugin_dir_path( PARSELY_FILE ) . 'build/admin-settings.asset.php';
+			$built_assets_url     = plugin_dir_url( PARSELY_FILE ) . '/build/';
+
+			// Admin settings JS.
```

Comment:
I think this comment and the one in line 100 are redundant.

Reply:
Removed both.